src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -417,6 +427,26 @@ def persist_flow_run(self, run_info: FlowRunInfo) -> None:
         )
         line_run_record.dump(self._run_infos_folder / filename)
 
+    def load_flow_run_info(self, line_number: int = None, parse_const_as_str: bool = False) -> FlowRunInfo:
+        json_loads = json.loads if not parse_const_as_str else json_loads_parse_const_as_str
+        flow_runs = []
+        for line_run_record_file in sorted(self._run_infos_folder.iterdir()):
```

Comment:
seems similar with load_node_run except the target deserialize class type, please try to extract them to a common function.

Reply:
Merged load_node_run_info and load_flow_run_info to a general function _load_run_info.

Reply:
Moved the part of json load to a new function _load_info_from_file. Considering not changing the result of load_detail, deserialize and resolve multimedia path are not included in _load_info_from_file.